sorc/ncep_post.fd/SCLFLD.f

```diff
@@ -1,42 +1,25 @@
 !> @file
```

Comment:
OK, the problem with all of these is you need two documentation blocks, and you have only one.

So you need this:

```
!> @file 
!> @brief This file contains some fuctions that do stuff.
!> @author Author O. File @date 1/1/22

!> Now a description of the subroutine not the file.
!> @param a param
!> @author Author O. Subroutine @date 1/1/22
subroutine ...
```

Note that there are two documentation blocks, separated by a fully-blank line. The first is for the file. The second is for the subroutine in the file.

Reply:
Hi Ed, thanks again for your comment. I made the changes and please have a look at my changes and let me if I made the changes you suggested.